functions/index.js

```diff
@@ -1,19 +1,26 @@
-const functions = require('firebase-functions')
-const admin = require('firebase-admin')
-const { Database } = require('arangojs')
+const functions = require("firebase-functions");
+const admin = require("firebase-admin");
+const { Database } = require("arangojs");
 
-admin.initializeApp()
+admin.initializeApp();
 
-const { arango } = functions.config()
+const { arango } = functions.config();
+const {
+  protocol = "http",
+  host,
+  port,
+  username,
```

Comment:
`user: username`

This is required unless also accompanied by updated instructions in `INSTALLATION.md` to change the function config var names. Actually, better to stick with the old names so as to not break existing installations.